src/GNDStk/Tree/src/top.hpp

```diff
@@ -0,0 +1,25 @@
+
+// -----------------------------------------------------------------------------
+// Tree::top()
+// See also: Tree::decl()
```

Comment:
No, they're different. Consider, for instance:

```
<?xml version="1.0" encoding="UTF-8"?>
<reactionSuite projectile="n" target="O16"
               evaluation="ENDF/B-8.0" format="1.9"
               projectileFrame="lab">
...
```

`decl()` ("declaration node") gives back a node named "xml" with two metadata.

`top()` ("top-level GNDS node") gives back a node named "reactionSuite" with five metadata (and some child nodes, not shown in the XML fragment above).

In case anyone wonders, I keep around the XML declaration node (if someone reads the tree from XML) because I believe it's a bad idea to throw away information. Imagine that someone wants to read a GNDS XML, fix some faulty data or perhaps add some new data, and then write it back out. If we take it upon ourselves to toss stuff out, like whatever might be in `<?xml...>`, then eventually someone is going to come ask why we don't leave stuff alone, and allow users to make the changes they want without us messing with other things.

Reply:
The code fragment above appears to be an older version. But the different between decl() and top() has always been the case.